backend/app/db/init.py

```diff
@@ -0,0 +1,16 @@
+import sqlite3
+from pathlib import Path
+
+from app.db.repositories.training_sessions import TrainingSessionsRepository
+
+
+def create_database(directory: Path) -> tuple[TrainingSessionsRepository]:
+    assert not directory.exists() or directory.is_dir(
+    ), "Database path has to be a folder or not exist"
```

Comment:
I'm not familiar with this annotation so leaving a comment to double-check if left on purpose